apps/site/components/Common/CodeBox/index.tsx

```diff
@@ -114,7 +114,12 @@ const CodeBox: FC<PropsWithChildren<CodeBoxProps>> = ({
           <span className={styles.language}>{language}</span>
 
           {showCopyButton && (
-            <Button kind="neutral" className={styles.action} onClick={onCopy}>
+            <Button
+              kind="neutral"
+              className={styles.action}
+              onClick={onCopy}
+              href="#"
```

Comment:
I've never heard of this being a question of accessibility. Do you have a WAI document that shows this?